core/config.js

```diff
@@ -16,28 +16,43 @@ module.exports = class Config {
   // Create config file if it doesn't exist.
   constructor (basePath) {
     this.configPath = path.join(basePath, 'config.json')
+    fs.ensureFileSync(this.configPath)
```

Comment:
It's weird that we ensure a file exists before ensuring that its parent directory exists.

Reply:
We don't ensure the parent directory exists later on. Besides, ensuring a file exists will ensure the whole path leading to it exists.